code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -708,7 +708,6 @@
 			druggy_dna.species.species_traits -= NOBLOODOVERLAY
 
 	invisible_man.update_body()
-	invisible_man.update_hair()
```

Comment:
Does the hair still get updated here?

Reply:
`update_body()` calls `update_body_parts()`